### ISSUE TITLE
Weird bug in autocomplete popup

### ISSUE THREAD

**Issue by tyctor** [gtimelog/gtimelog#250]
Hi

first, thanks for your application, i am using it few years, and it works well.

before month i traversed to new laptop, my system is Gentoo, Gnome (Wayland)

what occurs is weird, and probably is not even related to `gtimelog`, but i dont know where to post it, so adding it here.

what going on is, that autocomplete popup is not showing, but when i move application window near to left side, it start showing.

i recorded video:
[Screencast from 2024-02-07 10-13-46.webm](https://github.com/gtimelog/gtimelog/assets/44854182/d197b8aa-52ff-4718-93ce-3f8b775bd3e6)

**Comment by mgedmin** [comment]
Yeah, this is super weird.  Are you using multiple monitors?  Is this a Wayland or an Xorg session?

The bug is probably either in GTK 3, or in Mutter.

**Comment by tyctor** [comment]
thanks for your reply

> Yeah, this is super weird. Are you using multiple monitors? Is this a Wayland or an Xorg session?

yes, multiple monitors, and this happens only on primary display (laptop), on external monitor autocomplete works all the time.

**Comment by mgedmin** [comment]
I have seen similar bugs (not necessarily in gtimelog) in the past where such popups show up on the wrong monitor.

**Comment by tyctor** [comment]
just question, dark theme is not working on my desktop because am using GTK4 and gtimelog GTK3 ?

**Comment by mgedmin** [comment]
Perhaps.  You could see how other GTK3 apps behave, and if gtimelog acts differently, please open a separate issue.

**Comment by nblock** [comment]
I see a similar behavior  on my box (Arch Linux, wayland, SwayWM).

The popup works normally when no external monitors are connected.

With two external monitors, the popup only works on the left monitor. No popup appears when gtimelog is running on the right monitor and its starts to work again as soon as I move gtimelog to the left monitor.
